jobserver/views/status.py

```diff
@@ -1,12 +1,49 @@
 from django.db.models import Count
 from django.db.models.functions import Lower
+from django.shortcuts import get_object_or_404
 from django.template.response import TemplateResponse
+from django.utils import timezone
 from django.views.generic import View
 
 from ..backends import show_warning
 from ..models import Backend
 
 
+class PerBackendStatus(View):
```

Comment:
My instinct would for these to be API calls, rather than regular views

Given they're will be hit a lot, doesn't seem worth rendering a template?

Typical form for status checks just plain text response. They should be lightweight and fast.

Having a human readable page is maybe also worthwhile, but shouldn't be the same url I think?

Reply:
That's a point, we could just leave the UI for backend statuses to the existing `/status/` page.  I'll switch these to empty responses (freshping only cares about status code) and put the debugging information on the cards on the main status page.